### PR TITLE
Improve BlockquoteSyntax

### PR DESCRIPTION
**Pull request by chenzhiguang**
Strictly improves the test coverage.

### CODE REVIEW

File: tool/gfm_stats.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
   "212": "strict",
   "213": "strict",
   "214": "strict",
-  "215": "fail",
+  "215": "loose",
```
Comment (chenzhiguang) [inline tool/gfm_stats.json:84]: It will be **strict** after #478 is merged

File: lib/src/block_syntaxes/blockquote_syntax.dart [hunk 2]
```diff
@@ -21,26 +23,41 @@ class BlockquoteSyntax extends BlockSyntax {
...
 
-    var encounteredCodeBlock = false;
     while (!parser.isDone) {
+      final currentLine = parser.current;
       final match = pattern.firstMatch(parser.current);
       if (match != null) {
-        final line = match[1]!;
-        childLines.add(line);
-        encounteredCodeBlock = indentPattern.hasMatch(line);
+        // A block quote marker consists of a `>` together with a optional
```
Comment (srawlins) [inline lib/src/block_syntaxes/blockquote_syntax.dart:30]: nit: "an optional"

**Review by srawlins — CHANGES_REQUESTED** [review]
Nice little fix! One nit.

File: lib/src/block_syntaxes/blockquote_syntax.dart (hunk 2 above)
Comment (chenzhiguang) [reply lib/src/block_syntaxes/blockquote_syntax.dart]: Fixed

**Comment by chenzhiguang** [comment]
Rebased

**Comment by kevmoo** [comment]
@chenzhiguang – please rebase on `master` – we now have `flutter_markdown` validation in place

If there are breaks there, we'll have to dig to see what's expected/intended.

**Comment by chenzhiguang** [comment]
Rebased

**Comment by chenzhiguang** [comment]
## Benchmark

### Before

Screenshot 2022-11-04 at 18 01 48

### After
Screenshot 2022-11-04 at 18 29 16

File: lib/src/test.dart [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import '../markdown.dart';
```
Comment (kevmoo) [inline lib/src/test.dart:1]: Should delete this, right @chenzhiguang ?

**Review by kevmoo — CHANGES_REQUESTED** [review]

**Review by kevmoo — APPROVED** [review]
:shipit:

**Review by srawlins — APPROVED** [review]
❤️

**Pull request merged**